packages/patternfly-react/less/compound-label.less

```diff
@@ -0,0 +1,42 @@
+@import 'patternfly/variables';
```

Comment:
It is assumed that this is already imported (for any of our less files to work).